### ISSUE TITLE
feedback from max

### ISSUE THREAD

**Issue by max-mapper** [paulcbetts/electron-manifesto#2]
I think the overall goals are good (making it easier to get started, solving common problems). However I think the current proposal reeks a bit too much of rails style scaffolding. 

If it was me doing it I would make a list of things that are important to complete the 'whole picture' of the ecosystem, end to end, e.g. app packaging, UI components, app updating, development tools and make a list along the lines of https://github.com/sindresorhus/awesome-nodejs except specifically for Electron.

The part I don't like about the proposal is bundling up all of that stuff into one frameworky build tool. IMO the stuff that Electron needs around it is a diverse ecosystem of tools, meaning there won't be a 'correct' way to do something for every use case that one central tool will be able to facilitate.

That being said, discovery is hard then things aren't monolithic. To solve this I've found if you have a prominently featured open source list of community developed tools then people have an incentive to create tools and send pull requests adding their tools to the list.

tl;dr I think simply curating a list of community built third-party tools, prominently featuring it, and encouraging users to build tools for other users will be easier to maintain and launch in the short term
